main.conf

```diff
@@ -4,7 +4,9 @@
   "password": "",
   "config_mode_switch_pin": 22,
   "dht22_pin": 23,
-  "measurement_interval": "1h",
+  "bme280_sda_pin": 21,
+  "bme280_scl_pin": 22,
```

Comment:
Pin 22 is already assigned in the `config_mode_switch_pin` parameter above. I've just updated it to 20 and added a note in the README that pins need to be verified.